style.css

```diff
@@ -1,489 +1,772 @@
 * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-    --gold-color: #d2b429;
-    --header-font: 'Cormorant Upright', 'Georgia', serif;
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+	--gold-color: #d2b429;
+	--header-font: "Old Standard TT", "Georgia", serif;
 }
 
 body {
-    background-color: black;
-    margin: 0;
-    font-family: 'Montserrat', sans-serif;
-    font-weight: 400;
-    font-size: 20px;
-    color: white;
+	background-color: black;
+	margin: 0;
+	font-family: "Montserrat", sans-serif;
+	font-weight: 400;
+	font-size: 20px;
```

Comment:
It could be easier to count rems if we used 10px here, however it's totally up to you, just a convenience thing. :)

Reply:
I thought that rems are based on the font size of the body element... :) Does that mean I don't have to set rem units?

Reply:
That's correct, they are based on the size of fonts on the body. :) It's just it may be easier to count based on 10 than 20 :D Not a great maths problem anyways. :) I tend to use 10px, so I know I only need to cut one 0, just a preference thing. :)

Reply:
If you did not set any font size on body, it'd default to 16px which could make maths a little bit more cumbersome. ;)